src/s1reader/s1_reader.py

```diff
@@ -56,7 +56,12 @@ def parse_polynomial_element(elem, poly_name):
 
     half_c = 0.5 * isce3.core.speed_of_light
     r0 = half_c * float(elem.find('t0').text)
-    coeffs = [float(x) for x in elem.find(poly_name).text.split()]
+
+    if elem.find(poly_name) is None:  # some data with old IPF version (e.g. 2.36)
```

Comment:
I would probably say "with IPF < 2.36"

Reply:
@vbrancat Thanks for taking a look at the PR.
What I currently know about the relationship between IPF version and the format difference (for AZ FM rate polynomial) is that the change has taken place somewhere >2.36 and <2.82 (by taking a look at the S1 data I've processed so far). I took a look at the three different versions of S1 Product specification, but was not able to find a clear answer about the exact IPF version from which the format has changed.
I've put more context and description to the comment.